vendor/github.com/prometheus/client_golang/prometheus/promhttp/BUILD.bazel

```diff
@@ -7,6 +7,7 @@ go_library(
         "http.go",
         "instrument_client.go",
         "instrument_server.go",
+        "option.go",
```

Comment:
This file should be added in this PR but don't see it

Reply:
Fixed!